<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>zwei_construct_win</class>
 <widget class="QWidget" name="zwei_construct_win">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1381</width>
    <height>665</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>510</y>
     <width>292</width>
     <height>112</height>
    </rect>
   </property>
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <widget class="QLabel" name="exposure_time">
      <property name="text">
       <string>exposure_time</string>
      </property>
     </widget>
    </item>
    <item row="1" column="0">
     <widget class="QLineEdit" name="exposure_time_link"/>
    </item>
    <item row="1" column="1">
     <widget class="QLineEdit" name="exposure_time_richt"/>
    </item>
    <item row="2" column="0">
     <widget class="QPushButton" name="read_param_link">
      <property name="text">
       <string>read_param_link</string>
      </property>
     </widget>
    </item>
    <item row="2" column="1">
     <widget class="QPushButton" name="read_param_richt">
      <property name="text">
       <string>read_param_richt</string>
      </property>
     </widget>
    </item>
    <item row="3" column="0">
     <widget class="QPushButton" name="set_param_link">
      <property name="text">
       <string>set_param_link</string>
      </property>
     </widget>
    </item>
    <item row="3" column="1">
     <widget class="QPushButton" name="set_param_richt">
      <property name="text">
       <string>set_param_richt</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>510</y>
     <width>144</width>
     <height>143</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QLabel" name="label_4">
      <property name="text">
       <string>img name</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="img_name"/>
    </item>
    <item>
     <widget class="QPushButton" name="grab_zwei">
      <property name="text">
       <string>grab_zwei</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="save">
      <property name="text">
       <string>save</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="sys_cal">
      <property name="text">
       <string>sys_cal</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QLabel" name="link">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>67</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>link</string>
   </property>
  </widget>
  <widget class="QLabel" name="richt">
   <property name="geometry">
    <rect>
     <x>690</x>
     <y>0</y>
     <width>67</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>richt</string>
   </property>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>630</y>
     <width>255</width>
     <height>27</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QRadioButton" name="for_cal">
      <property name="text">
       <string>for_cal</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="set_param">
      <property name="text">
       <string>set_param</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="load_param">
      <property name="text">
       <string>load_param</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
